examples/cloud-router-2-virtual-device-connection/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "module_output" {
```

Comment:
Let's name this "FCR_VD_connection". It may be used for other resources in a combination of configs.